Makefile

```diff
@@ -432,9 +429,9 @@ endif
 
 .PHONY: install
 # target: install- install CouchDB :)
-install:
+install: release
 	@echo
-	@echo "Notice: There is no 'make install' command for CouchDB 2.x."
+	@echo "Notice: There is no 'make install' command for CouchDB 2.x+."
```

Comment:
maybe add a line that explains “we’ve run 'make release' here for you”

Reply:
will do on final pass